lib/Autocomplete.js

```diff
@@ -330,6 +330,12 @@ let Autocomplete = React.createClass({
     this._ignoreClick = false
   },
 
+  composeEventHandlers (internal, external) {
```

Comment:
I could have done something fancy here, like
```js
composeEventHandlers (...handlers) {
  return e =>
    handlers
      .filter(v => v)
      .forEach(handler => handler(e))
}
```

...but I decided to optimise for the actual use case we have now, which is that the first is always present, while the second is unknown.